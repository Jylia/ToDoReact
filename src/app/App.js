import React from 'react';
import TaskList from './components/task-list';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.muiTheme = getMuiTheme();
  }

  render() {
    const style = {
      margin: 24,
    };

    return (
      <MuiThemeProvider muiTheme={this.muiTheme}>
        <div className="App">
          <div className="App-header">
            <h2 style={style}>ToDo Lists</h2>
          </div>
          <div className="App-intro">
            <TaskList/>
          </div>
        </div>
      </MuiThemeProvider>
    );
  }
}

export default App;
