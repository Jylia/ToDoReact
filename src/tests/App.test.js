import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from '../app/App';
import { database } from 'firebase';
import {
  initializeApp
} from 'firebase';

const config = {
  apiKey: "AIzaSyCCYaZZ623P_pQu276vuGf761kZk_EhOBI",
  authDomain: "todo-85f3f.firebaseapp.com",
  databaseURL: "http://todo-85f3f.firebaseio.com",
  storageBucket: "todo-85f3f.appspot.com",
  messagingSenderId: "1034314432668"
};

initializeApp(config);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('renders tasks list', () => {
  const component = renderer.create(
    <div className="TaskList"></div>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
