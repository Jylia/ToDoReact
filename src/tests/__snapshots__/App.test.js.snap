exports[`test renders tasks list 1`] = `
<div
  className="TaskList" />
`;
